scale/storage/serializers.py

```diff
@@ -112,7 +91,7 @@ class ScaleFileBaseSerializerV5(ModelIdSerializer):
     media_type = serializers.CharField()
     file_type = serializers.CharField()
     file_size = serializers.IntegerField()  # TODO: BigIntegerField?
-    data_type = DataTypeField()
+    data_type = serializers.ListField(child=serializers.CharField(), source='data_type_tags')
```

Comment:
Is there a reason this was left as `data_type` instead of `data_type_tags`?

Reply:
It's pointless as v5 is going away but I kept it the field name the same for the v5 serializer out of habit.

Reply:
The v6 field has the name change.